sql/_27_banana_qa/issue_12053_show_index_cardinality/answers/show_index.answer

```diff
@@ -13,9 +13,11 @@
 ===================================================
 0
 ===================================================
+0
+===================================================
 Table    Non_unique    Key_name    Seq_in_index    Column_name    Collation    Cardinality    Sub_part    Packed    Null    Index_type    Func    Comment    Visible    
-dba.foo     1     idx1     1     a     A     3     null     null     YES     BTREE     null     null     YES     
-dba.foo     1     idx2     1     b     A     3     null     null     YES     BTREE     null     null     YES     
+dba.foo     1     idx1     1     a     A     6     null     null     YES     BTREE     null     null     YES     
```

Comment:
Why did the results change?

Reply:
Before when the ddl statement was executed only specified index statistics were updated. (ex: create index idx3 on foo (c); ) 
After maintaining the result need to put 'update statistics on foo' after ddl is executed.
But 'update statistics' statement updates all statistics in the specified table. So cardinality is updated as containing all the inserted data.

Reply:
Okay.